R/gibbs_draws.R

```diff
@@ -409,7 +409,35 @@ draw_R <- function(epsilon, incid, lambda, priors,
   R[t, ] <- R_fill
   R
 }
-
+##' Index before which at most a given probability
+##' mass is captured
+##'
+##' Across a matrix of discretised probability distributions
+##' (see \code{estimate_joint}
+##' this function returns the largest index
+##' (across all columns) such that the
+##' cumulative probability mass before index is
+##' 1 - \code{miss_at_most}.
+##'
+##'
+##' @inheritParams estimate_joint
+##' @param miss_at_most numeric. probability mass in the tail of the SI distribution
+##' @return
+##' @author Sangeeta Bhatia
+##' @export
+compute_si_cutoff <- function(si_distr, miss_at_most = 0.05) {
+  if (sum(si_distr) != 1) {
```

Comment:
this suggests that si_distr is a vector here i.e. for a single variant, so I don't think you can inheritParams and you need to write a help for the si_distr param?

Reply:
actually a couple of lines below you use colSums; so not sure which is the correct one but they don't seem consistent with one another?